tests/manage/pv_services/test_resource_deletion_during_pod_pvc_deletion.py

```diff
@@ -295,22 +302,33 @@ def setup_base(
         """
         Create PVCs and pods
         """
+        access_modes = [constants.ACCESS_MODE_RWO]
+        if interface == constants.CEPHFILESYSTEM:
+            access_modes.append(constants.ACCESS_MODE_RWX)
         pvc_objs = multi_pvc_factory(
             interface=interface,
             project=None,
             storageclass=None,
             size=self.pvc_size,
-            access_mode=constants.ACCESS_MODE_RWO,
+            access_modes=access_modes,
             status=constants.STATUS_BOUND,
             num_of_pvc=self.num_of_pvcs,
             wait_each=False
         )
 
         pod_objs = []
+
+        # Create one pod using each RWO PVC and two pods using each RWX PVC
         for pvc_obj in pvc_objs:
+            if pvc_obj.access_mode == constants.ACCESS_MODE_RWX:
```

Comment:
Why is there a condition here? The call to pod_factory looks the same for RWX and other access modes

Reply:
That is to create two pods if PVC is having RWX access mode